.github/workflows/build-and-publish-images.yml

```diff
@@ -19,11 +19,9 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Compute matrix
+      - name: Compute Matrix
         id: compute-matrix
-        run: |
-          MATRIX=$(ci/compute-mx.sh)
-          echo "MATRIX=${MATRIX}" | tee -a ${GITHUB_OUTPUT}
+        uses: ./.github/actions/compute-matrix
```

Comment:
the `computer-matrix` composite action checks out the code in addition to the steps here.

this should be deduped.